src/xrEngine/main.cpp

```diff
@@ -324,7 +324,7 @@ ENGINE_API void Startup()
     Device.WaitEvent(spatialCreated);
 
     g_pGamePersistent = dynamic_cast<IGame_Persistent*>(NEW_INSTANCE(CLSID_GAME_PERSISTANT));
-    R_ASSERT(g_pGamePersistent);
+    R_ASSERT(g_pGamePersistent || Engine.External.SkipGameModuleLoading());
```

Comment:
Just to be explicit =))
```suggestion
    R_ASSERT(g_pGamePersistent || Engine.External.CanSkipGameModuleLoading());
```